identity/users_test.go

```diff
@@ -39,6 +42,111 @@ func MockIdentityClient() *identity.IdentityClient {
 	}
 }
 
+func TestAccUser_ChangeUserPassword(t *testing.T) {
+	userLogin := testutils.RandPrefixString("TestAccUser", 32)
+	userEmail := fmt.Sprintf("%s@example.com", userLogin)
+	userPassword := testutils.RandString(32)
+	userNewPassword := testutils.RandString(32)
+
+	// Holds newly created user.
+	var newUser *identity.User
+
+	testutils.AccTest(t, testutils.TestCase{
+		Steps: []testutils.Step{
+
+			&testutils.StepClient{
+				StateBagKey: "user",
+				CallFunc: func(config *triton.ClientConfig) (interface{}, error) {
+					return identity.NewClient(config)
+				},
+			},
+
+			&testutils.StepAPICall{
+				StateBagKey: "user",
+				CallFunc: func(client interface{}) (interface{}, error) {
+					c := client.(*identity.IdentityClient)
+					ctx := context.Background()
+					input := &identity.CreateUserInput{
+						Email:    userEmail,
+						Login:    userLogin,
+						Password: userPassword,
+					}
+					user, err := c.Users().Create(ctx, input)
+					newUser = user
```

Comment:
Remove this and utilize the `stateBag` argument within your `CleanupFunc` implementation. More below...

Reply:
Will do.

I decided to leverage the variable I had already (see comment above), but I state bag is the way to go, indeed.

Changes coming right up! :)

Reply:
whyyyyyyyyyy did you put an @ before my name? it doesn't even make any sense